### PR TITLE
Introduce function to determine watercrafts

### LINKED ISSUE #35: Update list of air vehicles and add sea vehicles
- [x] add new helicopter: model id 469 (Sparrow)
- [x] add sea vehicles (only boats)
- [x] exclude sea vehicles from speed cameras

See also pull requests #29 and #30 for this.

### PR DESCRIPTION
**Pull request by Locky0**
Function renamed and added function to get if a vehicle is boat.

### CODE REVIEW

File: pawno/include/PPC_Common.inc [hunk 1]
```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
 }
 
 // This function returns "1" if the given vehicle-id is a plane or helicopter
-IsVehicleAirVehicle(vid)
+IsVehicleAirModel(vid)
```
Comment (CodeAlDente) [inline pawno/include/PPC_Common.inc]: Please change function name back to `IsVehicleAirVehicle` to follow naming convention.

File: pawno/include/PPC_Common.inc [hunk 2]
```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
...
 {
 	switch (GetVehicleModel(vid))
 	{
 		case 592, 577, 511, 512, 593, 520, 553, 476, 519, 460, 513, 548, 425, 417, 487, 488, 497, 563, 447, 469: return 1;
-		default: return 0;
```
Comment (CodeAlDente) [inline pawno/include/PPC_Common.inc:249]: Please undo this change as  `default` is the correct way to be used inside a `switch`.

File: pawno/include/PPC_Common.inc [hunk 3]
```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
...
 	}
-
+	return 0;
```
Comment (CodeAlDente) [inline pawno/include/PPC_Common.inc:251]: You can remove that as  `default` already returns zero if no  `case` has been called.

File: pawno/include/PPC_Common.inc [hunk 4]
```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
...
+}
+IsVehicleBoatModel(vid)
```
Comment (CodeAlDente) [inline pawno/include/PPC_Common.inc]: Please use `IsVehicleSeaVehicle` as function name to follow naming convention.

File: pawno/include/PPC_Common.inc [hunk 5]
```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
...
-		default: return 0;
 	}
-
+	return 0;
+}
+IsVehicleBoatModel(vid) 
+{
+	switch(GetVehicleModel(vid)) 
+	{
+		case 430,446,452,453,454,472,73,484,493,595: return 1;
```
Comment (CodeAlDente) [inline pawno/include/PPC_Common.inc]: Please fix vehicle model id `73` to `473`

File: pawno/include/PPC_Common.inc [hunk 6]
```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
...
+	}
```
Comment (CodeAlDente) [inline pawno/include/PPC_Common.inc:259]: Please use `default` here as well. Place the following right under `case`:

`default: return 0;`

File: pawno/include/PPC_Common.inc [hunk 7]
```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
...
 	return 0;
```
Comment (CodeAlDente) [inline pawno/include/PPC_Common.inc:261]: As we use `default`, this line can be removed.

**Review by CodeAlDente — CHANGES_REQUESTED** [review]
Thank you! Please see ny comments and apply the changes in order for an approval of your pull request.

File: pawno/include/PPC_Common.inc (hunk 1 above)
Comment (Yousha) [reply pawno/include/PPC_Common.inc]: or maybe `IsAirVehicle`...

**Review by CodeAlDente — APPROVED** [review]
